platforms/core-execution/snapshots/src/main/java/org/gradle/internal/snapshot/AbstractListChildMap.java

```diff
@@ -32,6 +32,16 @@ public boolean isEmpty() {
         return false;
     }
 
+    @Override
+    public int size() {
+        return entries.size();
+    }
+
+    @Override
+    public <RESULT> RESULT withNode(VfsRelativePath targetPath, CaseSensitivity caseSensitivity, NodeHandler<T, RESULT> handler) {
+        return null;
```

Comment:
❓ What is use of this method? Should we throw UnsupportedException here?

Reply:
Absolutely nothing, I guess I hit "implement methods" in IntelliJ and accidentally left it in. Well spotted, removed.